i18n/en.json

```diff
@@ -5234,6 +5234,10 @@
     "id": "oauth.gitlab.tos.error",
     "translation": "GitLab's Terms of Service have updated. Please go to gitlab.com to accept them and then try logging into Mattermost again."
   },
+  {
+    "id": "plugin.message_will_be_posted.dismiss_post",
```

Comment:
Why is this included here? I thought this is just an arbitrary string. I suppose this should be in the webapp instead.